bach/bach/dataframe.py

```diff
@@ -45,6 +45,7 @@
 #     - dict of axis labels -> functions, function names or list of such.
 
 Level = Union[int, List[int], str, List[str]]
+Scalar = Optional[Union[int, float, str, UUID]]
```

Comment:
```suggestion
Scalar = Union[int, float, str, bool, date, datetime, time, UUID]
```

This should work with date etc. too right?

I realize that my suggestion is not exhaustive either, but seems nice to have the python types for the 'basic' series at least. 
Also I realize that  `bool` is technically included already as it's a subtype of `int`, but I think it's nice to be explicit

Reply:
it sorta should. I'd just copied the types supported in `const_to_series` since that's what we call on Series.fillna